src/components/Navigation.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 const Navigation = () => {
   return (
     <div>
       <div className="App">
         <h1>React Router Mini</h1>
         <div>
-          <a href="">Home</a>
+          <Link to="/" isActive={(match,location) => {
+            return match.url === "/";
+          }}>Home</Link>
```

Comment:
`isActive` is a function that is a property of the `NavLink` component and isn't available to `Link`